browser-extension/src/displaySearchRecommendations.js

```diff
@@ -0,0 +1,46 @@
+(async () => {
+  const bannerUrl = chrome.extension.getURL('./models/banner/Banner.js'),
+    TournesolRecommendationsOptionsUrl = chrome.extension.getURL(
+      './models/tournesolRecommendations/TournesolRecommendationsOptions.js'
+    ),
+    TournesolSearchRecommendationsUrl = chrome.extension.getURL(
+      './models/tournesolRecommendations/TournesolSearchRecommendations.js'
+    );
+
+  const [
+    { Banner },
+    { TournesolRecommendationsOptions },
+    { TournesolSearchRecommendations },
+  ] = await Promise.all([
+    import(bannerUrl),
+    import(TournesolRecommendationsOptionsUrl),
+    import(TournesolSearchRecommendationsUrl),
+  ]);
+
+  const tournesolSearchRecommendationsOptions =
+    new TournesolRecommendationsOptions(
+      3,
+      1,
+      new Banner(),
+      '.style-scope.ytd-page-manager #container',
+      true
+    );
+  const tournesolSearchRecommendations = new TournesolSearchRecommendations(
+    tournesolSearchRecommendationsOptions
+  );
+
+  const process = () => {
+    if (location.search.includes('tournesolSearch')) {
+      chrome.storage.local.set({ searchEnabled: true });
+    }
```

Comment:
I don't remember the use case of this parameter.

Reply:
This parameter is to check if the search works well in the tests

Reply:
I was able to directly set during the tests the key `searchEnabled` to `true` in the local storage. It allowed me to delete the condition `location.search.includes('tournesolSearch')` but since I merged `main` into this branch it doesn't work anymore.

I'm checking what's happening.